CONTRIBUTING.md

```diff
@@ -16,6 +16,8 @@ All contributions to the repository must be submitted under the terms of the [Ap
 
 By contributing to this project, you agree to the Developer Certificate of Origin (DCO). This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution. See the [DCO](DCO) file for details.
 
+Use the `-s` or `--signoff` flag to signoff on a Git commit. See the [Git documentation](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s) for details.
```

Comment:
Do we have/need instructions on how to signoff earlier commits?

I think also this:

> Use the `-s` or `--signoff` flag to signoff on a Git commit.

could add a few details for clarity, similar to the [cert-manager docs](https://cert-manager.io/docs/contributing/sign-off/) (since I know I didn't understand all this until reading up on it):

> Use the `-s` or `--signoff` flag to indicate your DCO sign off for each commit. (See the [Git documentation](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s) for details.) This will add this line to the commit:
>```
>Signed-off-by: Contributor <contributor@example.com>
>```

Reply:
done as suggested for clarity and how to fix previous commits.